lib/connectors/kv-memory.js

```diff
@@ -201,6 +201,12 @@ KeyValueMemoryConnector.prototype.disconnect = function(callback) {
   process.nextTick(callback);
 };
 
+KeyValueMemoryConnector.prototype.flush =
+function(modelName, options, callback) {
+  this._store = Object.create(null);
```

Comment:
I forgot to submit my comments yesterday. Based on our last discussion, this comment is no longer relevant, as `flush` will be deleting only keys of a single model, i.e. the implementation will be different.